local-tests/test_multiple_restarts.py

```diff
@@ -33,49 +34,40 @@
 
 chain.set_flags_validator('validator')
 
-print('Starting the chain')
+printt('Starting the chain')
 chain.start('aleph')
 
-for run_duration, stop_duration, catch_up_duration in [[150, 20, 30], [20, 15, 30], [20, 15, 30]]:
-    print(f'Waiting {run_duration}s')
-    sleep(run_duration)
-
-    finalized_before_kill_per_node = check_finalized(chain)
-    print('Killing one validator and one nonvalidator')
+printt('Waiting for finalization')
+chain.wait_for_finalization(0)
+printt('Waiting for authorities')
+chain.wait_for_authorities()
 
+for sleep_duration in [21, 37, 15]:
+    printt('Killing one validator')
     chain[3].stop()
-    chain[4].stop()
+    finalized_before_kill = check_finalized(chain)
 
-    print(f'Waiting {stop_duration}s')
-    sleep(stop_duration)
+    printt(f'Waiting {sleep_duration}s')
+    sleep(sleep_duration)
 
-    print('Restarting nodes')
-    finalized_before_start_per_node = check_finalized(chain)
+    finalized_before_start = check_finalized(chain)
 
     # Check if the finalization didn't stop after a kill.
-    # Use a reduced rate since 1/4 nodes are offline, and fudge by 0.9 to give a margin of error.
-    if finalized_before_start_per_node[0] - finalized_before_kill_per_node[0] < stop_duration * (3/4) * 0.9:
-        print('Finalization stalled')
+    if finalized_before_start[0] - finalized_before_kill[0] < 10:
+        printt('Finalization stalled')
         sys.exit(1)
 
-    chain.start('aleph', nodes=[3, 4])
-
-    print(f'Waiting {catch_up_duration}s for catch up')
-    sleep(catch_up_duration)
-    finalized_after_catch_up_per_node = check_finalized(chain)
+    printt('Restarting nodes')
+    chain[3].start('aleph')
```

Comment:
let's extract all `aleph`s to `chain-name`, just to be explicit about a single chain :P

Reply:
'aleph' is just the name of the logfile in this context. I can imagine someone working on this test failing who would like to use different names for different stages.